CHANGELOG.md

```diff
@@ -1,5 +1,45 @@
 # Changes in IPython kernel
 
+## 6.0.0
+
+IPykernel 6.0 is the first major release in about two years, that brings a number of improvements code cleanup and new
+features to IPython.
+
+You should be able to view all closed issues and merged Pull Request for this
+milestone [on
+GitHub](https://github.com/ipython/ipykernel/issues?q=milestone%3A6.0+is%3Aclosed+),
+as for any major releases, we advise greater care when updating that for minor
+release and welcome any feedback (~50 Pull-requests).
+
+IPykernel 6 should contain all changes of the 5.x series, in addition to the
+followings non-exhaustive changes.
+
+
+ - The control channel on IPykernel 6.0 is ran in a separate thread, this may
+   change the order in which messages are process, though this change was necessary
+   to accommodate the debugger.
+
+ - Support for the debugger protocol, when using jupyter lab, retrolab or any
```

Comment:
Maybe swap this bullet with the first to lead with the feature?

Reply:
Thanks, done and added a couple other things. I'll probably send more updates in subsequent PRs there are still a couple of deprecations to remove.